screens/ContactsScreen.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { Text, View, Image, KeyboardAvoidingView } from 'react-native';
+import { List, Modal, Provider, Button } from '@ant-design/react-native';
+import PhoneInput from 'react-native-phone-input';
+import styles from '../styling/ContactsScreen.styles';
+import ListItem from '../components/ListItem';
+
+const Item = List.Item;
+
+class ContactsScreen extends Component {
+	async componentDidMount() {}
+	state = { phoneNumber: null };
```

Comment:
Setting your components default state should not happen in componentDidMount. If you want to change your state use ` setState()` in componentDidMount. Default state should be declared in the constructor for class components and ` useState(0)` for functional components. You can google it for more info about why. 😛